app/controllers/analytics_controller.rb

```diff
@@ -41,4 +40,23 @@ def index
   # rubocop:enable Metrics/MethodLength
   # rubocop:enable Metrics/LineLength
   # rubocop:enable Metrics/AbcSize
+
```

Comment:
It seems that these four methods could be in a concern, but this would be for SOP purposes, not because the concern would be used in multiple controllers.